articles/iot-dps/quick-create-simulated-device-tpm.md

```diff
@@ -220,13 +220,15 @@ In this section, you'll build and run the TPM simulator. This simulator listens
 
     ```cmd/sh
     npm install node-gyp -g
+    #for versions of Node.js <= version 10
     npm install ffi -g
+    #for versions of Node.js > version 10
+    npm install ffi-napi -g
```

Comment:
What's the user experience if they run this command to install both ffi and ffi-napi? Do they have to pick one or the other? If that's the case, then I would comment out one of the lines.

Reply:
They should choose either of those depending on their Node.js version. Today I was trying to install the "ffi" package on Node.js version 16 and it kept throwing errors and it turned out that the "ffi" package is not compatible with Node.js version 16 (in fact it's only compatible with Node.js versions up to 10) . So for versions of Node.js newer than 10 the "ffi-napi" package should be installed instead.

Reply:
Ok, then I would comment out line 224. People are going to copy and paste this codeblock without reading it critically. It's good to have both versions of the command in there but only one should be active at a time. Have the one that applies to the newer version be the default, and someone running an older version will have to uncomment their version.